
  To fix the following compile error by enclosing it in ifndef __ASSEMBLY__/endif.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
include/asm-generic/bug.h
include/asm-generic/bug.h: Assembler messages:
include/asm-generic/bug.h:7: Error: Unrecognized opcode: `extern'
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

Signed-off-by: Tsutomu Owa <tsutomu.owa@toshiba.co.jp>
-- owa

---
 include/asm-generic/bug.h |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-2.6.23.1-rt5/include/asm-generic/bug.h
===================================================================
--- linux-2.6.23.1-rt5.orig/include/asm-generic/bug.h
+++ linux-2.6.23.1-rt5/include/asm-generic/bug.h
@@ -3,7 +3,9 @@
 
 #include <linux/compiler.h>
 
+#ifndef __ASSEMBLY__
 extern void __WARN_ON(const char *func, const char *file, const int line);
+#endif /* __ASSEMBLY__ */
 
 #ifdef CONFIG_BUG
 
